config/rbac/cluster_role_binding.yaml

```diff
@@ -9,5 +9,5 @@ roleRef:
   name: manager-role
 subjects:
 - kind: ServiceAccount
-  name: default
+  name: nexus-operator-sa
```

Comment:
the `ServiceAccount` should simply have `name: sa` here since `kustomize` will adjust it

Reply:
I thought I tried that and it didn't adjust it - let me try once more to make sure I didn't miss something

Reply:
Tried again with kustomize v3.9.2, and the resulting ClusterRoleBinding and RoleBindings in the generated bundle manifests have just `sa` as their subject name. In addition, the CSV has no `clusterPermissions` or `permissions`.

kustomize isn't my strongest area ... what am I missing?

Reply:
Ahh, I see the problem. You should move your `ServiceAccount` definition into its own file in the `config/rbac` directory and then add that file to `config/rbac/kustomization.yaml`. See https://github.com/RedHatGov/gitea-operator/blob/main/config/rbac/service_account.yaml for an example

Reply:
Ah, that's better :) Ok, thanks! Let me correct in all four PRs and I'll let you know when they're done.